### PR TITLE
fixes and update of readme for v0.9 release

### PR DESCRIPTION
**Pull request by rsnancollas**
Made a few changes to support the 0.9 release

### CODE REVIEW

File: support-docs/README.md [hunk 1]
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
...
+|tolerance|nominal tolerance of a resistor|definitions.json#/unit| |
+|powerRating|measure of power a resistor can dissipate indefinitely without degrading performance|definitions.json#/unit| |
+|temperatureCoefficient|change in resistance when the temperature is changed|Number| |
+|maxOverloadVoltage|maximum voltage that can be applied to the resistor for a short period of time|definitions.json#/unit| |
+|maxLimitingElementVoltage|maximum voltage value that can be applied continuously to the resistor|definitions.json#/unit| |
+|minTemperature|minimum temperature under which a resistor can be expected to reliably operate|definitions.json#/unit| |
+|maxTemperature|maximum temperature under which a resistor can be expected to reliably operate|definitions.json#/unit| |
+|resistorDerating|graph object to capture resistance changes with temperature|definitions.json#/graphDefiniton| |
+|package|package size of resistor|graph.json#/package| |
+
```
Comment (asadate) [inline support-docs/README.md:272]: type: "definitions.json#/package", not "graph.json..."

File: part-spec/graph.json [hunk 2]
```diff
@@ -1,61 +1,65 @@
...
-            "label": {
-                "description": "description of the data in a curve",
-                "type": "string"
-            },
-            "xData": {
-                "description": "x value of data being plotted",
-                "type": "array",
-                "items": {
+    "$id": "https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/Registers.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
```
Comment (asadate) [inline part-spec/graph.json:3]: "/part-spec/graph.json" not "Registers.json"

File: support-docs/README.md [hunk 3]
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
...
+|:----|:----|:----|:----|
+|length|length of a side of a package|#/$definitions/unit| |
+|width|width of a side of a package|#/$definitions/unit| |
+|height|height of a package|#/$definitions/unit| |
+|nominalFootprints|references to external footprints in standard CAD formats|array of #/$definitions/externalFile| |
+|breakoutExamples|references to external board file that contains layout breakout example|array of #/$definitions/externalFile| |
+|partModelInformation|reference to an external XML file that contains a part model in IPC/DAC2552 format|#/$definitions/externalFile| |
+|standardPackageSize|name of standard package size (imperial)|Must set either Ref or Type| |
+|standardPackageType|name of standard package types|Must set either Ref or Type| |
+
```
Comment (asadate) [inline support-docs/README.md:130]: standardPackageType & standardPackageSize: "do we want to add string as a type"?

File: support-docs/README.md [hunk 4]
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
...
-|:----|:----|:----|:----|
-|componentID|common component identifying information, such as mpn|componentID|Yes|
+####  3.4.14	 Value
+
+|Property|Description|JSON Data Type|Required?|
+|:----|:----|:----|:----|
+|typValue|typical value|Number| |
+|minValue|minimum value|Number| |
+|maxValue|maximum value|Number| |
+
```
Comment (asadate) [inline support-docs/README.md:210]: It looks like we forgot to remove "value" when we merged it with "unit".

File: support-docs/README.md [hunk 5]
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
...
+|value|capacitor value|definitions.json#/unit|Yes|
+|tolerance|nominal tolerance of a capacitor|definitions.json#/unit| |
+|ratedVoltage|maximum voltage which may be applied continuously to a capacitance|definitions.json#/unit| |
+|dielectric|dielectric material used in the capacitor|String| |
+|polarized|describes whether the capacitor is polarized|Boolean| |
+|equivalentSerieResistance|equivalent series resistance (ESR) of the capacitor|definitions.json#/unit| |
+|temperatureCoefficient|change in capacitance when the temperature is changed|Number| |
+|minTemperature|minimum temperature under which a capacitor can be expected to reliably operate|definitions.json#/unit| |
+|maxTemperature|maximum temperature under which a capacitor can be expected to reliably operate|definitions.json#/unit| |
+|capacitorDerating|graph object to capture capacitance changes with voltage|definitions.json#/graphDefiniton| |
```
Comment (asadate) [inline support-docs/README.md]: Should be"graph.json#/graphDefinition".

File: support-docs/README.md [hunk 6]
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
...
+|tolerance|nominal tolerance of an inductor|definitions.json#/unit| |
+|ratedCurrent|maximum continuous current the inductor can handle|definitions.json#/unit| |
+|saturationCurrent|current where the inductor enters the magnetic state, and the inductance drops a specified amount|definitions.json#/unit| |
+|rmsCurrent|DC current that produces an inductor temperature rise of 40 degrees Celsius|definitions.json#/unit| |
+|selfResonantFrequency|frequency at which the inductor becomes a capacitor|definitions.json#/unit| |
+|dcResistance|DC resistance of the inductor|definitions.json#/unit| |
+|temperatureCoefficient|change in inductance when the temperature is changed|Number| |
+|minTemperature|minimum temperature under which a inductor can be expected to reliably operate|definitions.json#/unit| |
+|maxTemperature|maximum temperature under which a inductor can be expected to reliably operate|definitions.json#/unit| |
+|saturationCurve|graph object to capture inductor saturation with current|definitions.json#/graphDefiniton| |
```
Comment (asadate) [inline support-docs/README.md]: graph.json#

File: support-docs/README.md [hunk 7]
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
...
+|resonantFrequencyCurve|graph object to capture inductor resonant frequency|definitions.json#/graphDefiniton| |
```
Comment (asadate) [inline support-docs/README.md]: Should be "graph.json#/graphDefinition".

File: support-docs/README.md [hunk 8]
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
...
+|package|package size of inductor|definitions.json#/package| |
+
+####  3.8.4	 Common Mode Choke
+
+|Property|Description|JSON Data Type|Required?|
+|:----|:----|:----|:----|
+|componentID|common component identifying information, such as mpn|definitions.json#/componentID|Yes|
+|diffModeCutoff|frequency at which the differential mode attenuation equals -3dB|definitions.json#/unit| |
+|commonModeAttenuation|graph object to capture common mode attenuation of a common mode choke at various frequencies|definitions.json#/graphDefiniton| |
```
Comment (asadate) [inline support-docs/README.md]: "graph.json#/graphDefinition". It looks like we have to make these  fixes in the source files (passive.json, power.json,..)

**Review by asadate — CHANGES_REQUESTED** [review]
Looks like the script is working well. Except for 1 or 2 issues, most of the issues I've flagged can be traced to the source file. Mostly around the location of "graphDefinition".  Let's fix the issues in the power.json and passive.json files before merging this version

File: support-docs/README.md (hunk 4 above)
Comment (rsnancollas) [reply support-docs/README.md:210]: I didn't make this change in this version - that's for 1.0, so I think it's okay we don't have this fixed yet.

File: support-docs/README.md (hunk 1 above)
Comment (rsnancollas) [reply support-docs/README.md:272]: Looks fixed.

File: part-spec/graph.json (hunk 2 above)
Comment (rsnancollas) [reply part-spec/graph.json:3]: Looks fixed.

File: support-docs/README.md (hunk 3 above)
Comment (rsnancollas) [reply support-docs/README.md:130]: good call.

File: support-docs/README.md (hunk 5 above)
Comment (rsnancollas) [reply support-docs/README.md]: looks fixed.

File: support-docs/README.md (hunk 6 above)
Comment (rsnancollas) [reply support-docs/README.md]: thanks for fixing.

File: support-docs/README.md (hunk 7 above)
Comment (rsnancollas) [reply support-docs/README.md]: looks done.

File: support-docs/README.md (hunk 8 above)
Comment (rsnancollas) [reply support-docs/README.md]: thanks for fixing this. I'll re-generate.

**Comment by rsnancollas** [comment]
RE-ran the script and updated the README.

**Review by asadate — APPROVED** [review]

**Pull request merged**